src/components/entity/update/row.vue

```diff
@@ -85,10 +85,7 @@ const resize = () => {
   textarea.value.resize();
   if (minHeightOutdated) setMinHeight();
 };
-const focus = () => textarea.value.focus();
-defineExpose({
-  textarea: { resize, focus }
-});
+defineExpose({ textarea: computed(() => ({ ...textarea.value, resize })) });
```

Comment:
We need to expose `el` from the textarea. But that's a little harder than it was for `resize()` and `focus()`, because `el` isn't a function: `textarea.value` is a reactive object, and `textarea.value.el` is the element (not a ref of the element). I decided to take a different approach, copying `textarea.value` and just overwriting the `resize()` function. That actually feels like a better approach in general than what it was before.